.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: python
-
 sudo: false
-
 python:
-  - '3.4'
-  - '3.5'
-  - '3.6'
-
+- '3.4'
```

Comment:
These list entries should be indented two spaces for readability.  I'm not sure if YML requires or or not, but it's the standard thing to do.